__tests__/Unit/Components/DevFeature/DevFeature.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { screen } from '@testing-library/react';
+import DevFeature from '@/components/DevFeature';
+
+import { renderWithRouter } from '@/test_utils/createMockRouter';
+
+describe('DevFeature', () => {
+    it('Should not render anything when there is no dev flag', () => {
+        renderWithRouter(
+            <DevFeature>
+                <p>Children</p>
+            </DevFeature>
+        );
+
+        expect(screen.queryByText('Children')).not.toBeInTheDocument();
+    });
+    it('Should not render anything when dev=true', () => {
```

Comment:
The test case title and the expected result do not comply with each other.

Reply:
Corrected